jobs/generation/MetaGenerator.groovy

```diff
@@ -240,28 +242,91 @@ repos.each { repoInfo ->
         folder(generatorFolder) {}
     }
 
+    // Create a view for all jobs in this folder that don't end with prtest
+    dashboardView("${projectFolder}/Official Builds") {
+        recurse()
+        jobs {
+            regex(/.*(?<!prtest)$/)
+        }
+
+        columns {
+            status()
+            weather()
+            name()
+            lastSuccess()
+            lastFailure()
+            lastDuration()
+        }
+
+        topPortlets {
+            jenkinsJobsList {
+                defaultName = "${projectFolder} jobs"
+            }
+        }
+
+        rightPortlets {
+            buildStatistics {
+                displayName 'Build Statistics'
+            }
+        }
+
+        def createPortletId = {
+            def random = new Random()
+            return "dashboard_portlet_${random.nextInt(32000)}"
+        }
+
+        configure { view ->
```

Comment:
Unfortunately, the portlets available in the dsl doesn't include some of the useful ones, so I'm configuring them manually. Values were taken from the config.xml on my private Jenkins server.